tuktuktravel/src/components/FormUsers.js

```diff
@@ -40,6 +41,7 @@ class FormUsers extends Component {
 
 
   render(){
+    console.log(this.state)
```

Comment:
Now that it's working, no need for console.log anymore :-)

Reply:
Done